packages/core/src/server/methods/mutation-test-method.ts

```diff
@@ -0,0 +1,98 @@
+import { createInjector } from 'typed-inject';
+import { MutantResult, PartialStrykerOptions } from '@stryker-mutator/api/core';
+import { commonTokens } from '@stryker-mutator/api/plugin';
+
+import { MutationTestExecutor } from '../../process/4-mutation-test-executor.js';
+import { provideLogger } from '../../di/provide-logger.js';
+import { PrepareExecutor } from '../../process/1-prepare-executor.js';
+import { MutantInstrumenterExecutor } from '../../process/2-mutant-instrumenter-executor.js';
+import { Stryker } from '../../stryker.js';
+import { BroadcastReporter } from '../../reporters/index.js';
+import { DryRunExecutor } from '../../process/3-dry-run-executor.js';
+import { coreTokens } from '../../di/index.js';
+import { ConfigError, retrieveCause } from '../../errors.js';
+import { LogConfigurator } from '../../logging/log-configurator.js';
+
+export class MutationTestMethod {
```

Comment:
This is a copy of the `Stryker` class. Why have 2 separate processes for the same thing? I think we should reuse the `Stryker` class.